BLSQ/iaso
# POLIO-926: campaign history logs some fields are not displayed

- budget log keys and values were missing 
- duplicated keys

Related JIRA tickets : [POLIO-926](https://bluesquare.atlassian.net/browse/POLIO-926?atlOrigin=eyJpIjoiYWViZTlhMzM5YzA1NDVkN2JhMWY0ZmQ1MDEyZGU2OWEiLCJwIjoiaiJ9)

## Self proofreading checklist


## Changes

-  add logic in `getLabel` to remove workflow suffix 'at_WFEDITABLE' for budget keys
- removed duplicated keys

## How to test

- Go to campaigns
- Choose a campaign
- Fill the dates in Budget tab
- Go to campaign log history and check if all fields are in the log as well as the values are well displayed and in good date format

[POLIO-926]: https://bluesquare.atlassian.net/browse/POLIO-926?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
